app/src/shared/routes.ts

```diff
@@ -0,0 +1,3 @@
+export const routes = {
+  processFiles: '/1.0/processFiles',
+}
```

Comment:
file needs a newline char.

Reply:
Thanks, I'll be modifying the linter in an upcoming PR to prevent this from getting through